src/main/kotlin/inc/evil/d17map/web/filters/RequestCachingFilter.kt

```diff
@@ -8,7 +8,7 @@ import org.springframework.core.annotation.Order
 import org.springframework.stereotype.Component
 import org.springframework.web.filter.OncePerRequestFilter
 
-@Order(Ordered.HIGHEST_PRECEDENCE)
+@Order(Ordered.HIGHEST_PRECEDENCE + 1) // +1 because we want the GlobalExceptionHandlingFilter to be executed first
```

Comment:
Potrzebujemy tego komentarza tu w sumie?

Reply:
Jak dla mnie to nie, chociaż jest swego rodzaju przypominajką 😅

Reply:
okej to zostawmy